### PR TITLE
Added missing link, fixed ordering of subheadings

### PR DESCRIPTION
**Pull request by drpayyne**
## Purpose of this pull request

This pull request (PR) adds the missing reference link to the Magnifier widget, fixes ordering of subheadings, and makes titles consistent.

## Affected DevDocs pages

-  https://devdocs.magento.com/guides/v2.3/javascript-dev-guide/widgets/jquery-widgets-about.html

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: src/guides/v2.3/javascript-dev-guide/widgets/jquery-widgets-about.md [hunk 1]
```diff
@@ -10,16 +10,17 @@ This guide discusses the following widgets:
 
 -  [Accordion widget]
 -  [Alert widget]
--  [Breadcrumb widget]
+-  [Breadcrumbs widget]
```
Comment (rogyar) [inline src/guides/v2.3/javascript-dev-guide/widgets/jquery-widgets-about.md]: Please, change the references at the bottom of the document correspondingly. It applies for all your modifications. 

Thank you!

Comment (drpayyne) [reply src/guides/v2.3/javascript-dev-guide/widgets/jquery-widgets-about.md]: Fixed. Thanks, @rogyar!

**Comment by dobooth** [comment]
running tests

**Pull request merged**